pkgs/development/libraries/libmsgpack/generic.nix

```diff
@@ -8,7 +8,12 @@ stdenv.mkDerivation rec {
 
   inherit src patches;
 
-  buildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake ];
+
+  crossAttrs = {
```

Comment:
I assume the idea here is to make it easier to add more later? I think it's cleaner to not leave hooks for future extensions.